code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1164,6 +1164,26 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		. = 1
 	return ..() || .
 
+/datum/reagent/consumable/ethanol/bureau
+	name = "Bureaucratic Perfection"
+	description = "Smells of freshly signed and filed papers.."
+	nutriment_factor = 1 * REAGENTS_METABOLISM
+	color = "#806029" // rgb: 128, 86, 41 (what is this used for?)
+	boozepwr = 20
+	quality = DRINK_GOOD
+	taste_description = "pure organization"
+	glass_icon_state = "beuroglass"
+	glass_name = "Bureaucratic Perfection"
+	glass_desc = "Smells of freshly signed and filed papers.."
+	name = "Bureaucratic Perfection"
```

Comment:
duplicate

Reply:
im sorry im confused on what this comment means exactly?

Reply:
You have "Name" modified twice in this.